app/src/main/java/com/github/geohunt/app/utility/Assertions.java

```diff
@@ -0,0 +1,13 @@
+package com.github.geohunt.app.utility;
+
+public class Assertions {
+    private Assertions() {
+
+    }
+
+    public static <T> void assertNonNull(T value) {
+        if (value == null) {
+            throw new AssertionError("The value should not be NULL");
+        }
+    }

```

Comment:
I believe that this method could be replaced by the standard `Objects.requireNonNull` method